docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,148 @@
+
+The Battery Historian tool takes the bug report file as an input to analyze and compare the stats. After processing the report data, the Battery Historian uses HTML format to visualize the stats in a web browser.
+
+To use the Battery Historian tool, ensure that you have the bug report files readily available. If you do not have the bug report file with, you can generate one using the following procedure:
+
+### Generate Bug Report File
+
+To generate a bug report file, follow these steps:
+
+1. From your Linux Terminal, enter the following commands:
+
+    ```
+
+    $] sdb root on
+
+    $] sdb shell
+
+    ```
+
+2. From your Tizen device terminal, enter the following commands:
+
+    ```
+
+    sh-3.2] dumpsys batterstats \&gt; /tmp/battery\_dump.txt
+
+    sh-3.2] exit
+
+    ```
+
+3. From Linux Terminal, enter the following commands::
+
+    ```
+
+    $] sdb pull /tmp/battery\_dump.txt
+
+    ```
+
+After you execute the commands, the `batterystats.txt` file is generated in the specified directory using the optional path argument.
+
+> [!NOTE]
+> If no path is specified, the file gets created in your home directory.
+
+### View and Analyze Battery Data
+
+To view and analyze your data, follow these steps:
+
+1. In Battery Historian Main Window, click **Browse**.
+
+2. Select the appropriate bug report file.
+
+3. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in the file. After the analysis is done, a battery stats chart appears:
+
+**Figure: Battery Historian Chart**
+
+![](./media/graph1.png)
+
+The Battery Historian chart graph illustrates the battery consumption events over time, where usage of the various device components correlates to battery consumption.
+
+#### Interpret the Graph
+
+ - The vertical axis represents the battery percentage, while the horizontal axis represents time.
+ - The sloping black bar indicates your battery percentage level throughout the day.
+ - The colored line segments represent the various apps, services, and other elements that have used battery life.
+   - For instance, a green bar denotes an active time for WiFi.
+ - You can add more metrics, by clicking on Add Metric and the corresponding metric will appear on the chart. Some of the available metrics are  Charging status, Voltage, Device active, Plug and so on.
+ - The chart generated is intuitive, hovering over any metric name in the first column shows you more information about that metric.
+
+**Figure: System Stats**
+
+![System Stats](./media/graph2.png)
+
+The **System Stats** tab includes overall system statistics, such as screen on time, WiFi idle time, and so on. It gives an overview of what is happening with your Tizen device. For example, you can analyze the impact of a particular application on the battery during its use.
```

Comment:
How about moving this line before the "System Stats" image, so that the introduction can come before the image as like below example?


The **System Stats** tab includes overall system statistics, such as screen on time, WiFi idle time, and so on. It gives an overview of what is happening with your Tizen device. For example, you can analyze the impact of a particular application on the battery during its use.

**Figure: System Stats**

_image_